acceptance-tests/src/test/js/failingStages.js

```diff
@@ -75,10 +74,12 @@ module.exports = {
     'Step 05' : function(browser) {
         const blueRunDetailPage = browser.page.bluePipelineRunDetail().forRun(JOB, 'jenkins', 1);
 
+        // Open resultitem
+        blueRunDetailPage.clickFirstResultItem();
```

Comment:
I think this breaks the test, or rather, makes it pass but with incorrect behavior. 
Should not need to click first result item to have it expanded, it should be auto expanded. Reverting this will have a correctly failing test, which shows what the behavior should be.

Reply:
Right, gotcha